x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -104,9 +104,10 @@ const NodeEventsListItem = memo(function ({
   nodeID: string;
   eventCategory: string;
 }) {
-  const timestamp = eventModel.eventTimestamp(event);
-  const eventID = eventModel.eventID(expandDottedObject(event));
-  const winlogRecordID = eventModel.winlogRecordID(event);
+  const expandedEvent = expandDottedObject(event);
```

Comment:
Since, this is an expensive (I reckon) calculation, don't we need to memoize this?

I see component is already memoized on the basis of props `event`, `nodeId` & `eventCategory`. But even if eventCategory changes, we do not want to recalculate, expandDottedObject, right? Or am i missing something here?

Other than that it looks good to me.

Reply:
This component displays a list of events of a certain category, so all events would rerender anyway when that prop changes.

Reply:
okay, got it Thanks.